src/tests/unit/tests/common/components/scanning-spinner/__snapshots__/scanning-spinner.test.tsx.snap

```diff
@@ -1,13 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ScanningSpinner renders isScanning = <false> 1`] = `null`;
+exports[`ScanningSpinner renders isScanning = <false> 1`] = `<DocumentFragment />`;
 
 exports[`ScanningSpinner renders isScanning = <true> 1`] = `
-<StyledSpinnerBase
-  aria-live="test-aria-live"
-  className="scanningSpinner"
-  label="test-label"
-  role="alert"
-  size={3}
-/>
+<DocumentFragment>
+  <div
```

Comment:
It'd be good to mock `Spinner` so this snapshot is closer to the old one.

Reply:
resolved the comment